src/app/diary-module/containers/sign-in/sign-in.component.scss

```diff
@@ -0,0 +1,91 @@
+#container {
```

Comment:
There's nothing wrong here but. If there's no special need, it's better to use classes for styling.